### PR TITLE
fix(Field.Affix): make more robust

### PR DESCRIPTION
**Pull request by eirikbacker**
Adjustments found while doing the canceled #2771

### CODE REVIEW

File: packages/css/field.css [hunk 1]
```diff
@@ -68,39 +68,45 @@
...
+  align-self: stretch; /* If ds-field is placed inside a flex container like ds-field, we need to fill width */
   background: var(--ds-color-neutral-background-subtle);
   border-radius: var(--ds-border-radius-md);
   box-sizing: border-box;
   color: var(--ds-color-neutral-text-subtle);
   display: inline-flex; /* Using inline-flex to match native inline-block behaviour of <input> */
-  position: relative;
+  max-width: 100%;
   white-space: nowrap;
   align-self: stretch; /* If ds-field is placed inside a flex container  we need to fill width */
```
Comment (Barsnes) [inline packages/css/field.css:83]:
```suggestion
```
This is here twice

File: packages/react/src/components/form/Field/FieldAffix.tsx [hunk 2]
```diff
@@ -2,24 +2,27 @@ import cl from 'clsx/lite';
...
 export type FieldAffixProps = Omit<HTMLAttributes<HTMLDivElement>, 'prefix'>;
 
 export const FieldAffix = forwardRef<HTMLSpanElement, FieldAffixProps>(
-  function FieldAffix(rest, ref) {
-    return <span aria-hidden='true' ref={ref} {...rest} />;
+  function FieldAffix({ className, ...rest }, ref) {
+    return (
+      <span
+        className={cl('ds-field-affix', className)}
+        aria-hidden='true'
```
Comment (Barsnes) [inline packages/react/src/components/form/Field/FieldAffix.tsx:22]: Should we put this after `rest` so we make sure the user can't turn this off?

File: packages/react/src/components/form/Field/Field.stories.tsx [hunk 3]
```diff
@@ -80,36 +86,75 @@ export const Preview: Story = (args) => {
 // @ts-expect-error ts2559: Preview.args uses more properties for testing than what is supported by <Field>
 Preview.args = toggles;
 
-export const Adornments: Story = () => (
+export const Affix: Story = () => (
```
Comment (Barsnes) [inline packages/react/src/components/form/Field/Field.stories.tsx:89]: You also need to update the MDX file. The story for `Field` is not working anymore.

**Review by Barsnes — DISMISSED** [review]

File: packages/css/field.css (hunk 1 above)
Comment (eirikbacker) [reply packages/css/field.css:83]: well spotted!

File: packages/react/src/components/form/Field/FieldAffix.tsx (hunk 2 above)
Comment (eirikbacker) [reply packages/react/src/components/form/Field/FieldAffix.tsx:22]: mm, I think if they really want to set `aria-hidden="false"` they are on their own and we should not prevent it being possible ☺️

Comment (Barsnes) [reply packages/react/src/components/form/Field/FieldAffix.tsx:22]: fair enough, makes sense to me

**Review by Barsnes — APPROVED** [review]
Implementation looks good, missing some docs - but that goes for all of `Field`, which we will do later on

**Pull request merged**